Better structured project (#19)
tomrf1/scala-school: corporate-connections/README.md

(Server started, use Enter to stop and go back to the console...)
```

### Task 1: proxy appointments data for an officer
Implement the `/officer/:id/appointmentsProxy` endpoint.

Pass through the unaltered Companies House API appointments response for an officerId.

E.g. for officerId `aEdQfmEjiBuB7tLwOP_Wfg-JA-8`: `https://api.company-information.service.gov.uk/officers/aEdQfmEjiBuB7tLwOP_Wfg-JA-8/appointments`

Build on the `CompaniesHouseService` class in the `services` package for talking to the Companies House API.

> ### Task 2: return company names for an officer

Reviewer: ```suggestion
### Task 2: return company ids for an officer
```

> ### Task 2: return company names for an officer

Reviewer: ```suggestion
### Task 2: return company numbers for an officer
```

Reviewer: Because it's company number in the [Companies House API](https://developer-specs.company-information.service.gov.uk/companies-house-public-data-api/reference/officers/list), and that's what we'll get back in our response from them, let's call it company number everywhere.

Reviewer: (Meanwhile, officer id is fine, because that's the Companies House convention)

Implement the `/officer/:officerId/appointments` endpoint.

Use Circe to decode the Companies House response into Scala case classes. Use the `OfficerAppointmentResponse` model in `models/CompaniesHouseModels.scala`.

> Then return just the list of companies as a JSON array.

Reviewer: ```suggestion
Then return just the list of company numbers as a JSON array.
```
